config.ru

```diff
@@ -0,0 +1,6 @@
+# This file is used by Rack-based servers to start the application.
+
```

Comment:
Kindly update your README to include descriptions about this project, how to set the project up in the terminal, and how to create and migrate your project database. Please make use of the Microverse README template and complete all the necessary sections.